docs/features/countdowns.md

```
@@ -0,0 +1,336 @@
+
+- **Creative Messages** - Whimsical, family-friendly messages
+- **Context-Aware** - Different messages for "today", "tomorrow", and future dates
+- **Daily Refresh** - Messages regenerate every 24 hours
+- **Automatic Caching** - Reduces API calls
+
+### Setup Gemini API (Optional)
+
+1. Visit [Google AI Studio](https://makersuite.google.com/app/apikey)
+2. Create a free API key
+3. Add to your `.env` file:
+   ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)